Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -965,6 +1081,8 @@ namespace PlayerCommands
 			PrintUserCmdText(client, L"|  list - show factory modules and build status");
 			PrintUserCmdText(client, L"|  clear <index> - clear build queue for factory module at <index>");
 			PrintUserCmdText(client, L"|  add <index> <type> - add item <type> to build queue for factory module at <index>");
+			PrintUserCmdText(client, L"|  pause <index> - pause factory module at <index>");

```

Comment:
should we move these down to below the type definitions that seem to be associated with the `add` doc above?

Reply:
mmm....
pause and resume look to be the best under "add" because they can be used only after "add" was applied
at the same time all commands look to be better placed at the top of instruction, keeping all commands easily foundable grouped at its beginning

btw I noticed they write | at phrase start to fix issue with phrase disappearance. I'll add the same to "autobuy" to have indents

Reply:
they can be under add, but right now they appear between add's main docs and the docs for one of its parameters. please put pause/resume after the docs for <type>

Reply:
if you insist